### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by billyjbryant**
# Description

Adding documentation on README & Contributing.

### CODE REVIEW

**Review by clarkemn — APPROVED** [review]

File: CODE_OF_CONDUCT.md
Comment (Harpo6) [inline CODE_OF_CONDUCT.md:1]: I’m

**Review by arifhosenbelal42-a11y — COMMENTED** [review]
Image

**Comment by arifhosenbelal42-a11y** [comment]
Same thing not cheng

**Review by arifhosenbelal42-a11y — COMMENTED** [review]
Image

**Pull request merged**